interop/client/client.go

```diff
@@ -39,6 +40,7 @@ var (
 	caFile                = flag.String("ca_file", "", "The file containning the CA root cert file")
 	useTLS                = flag.Bool("use_tls", false, "Connection uses TLS if true")
 	useALTS               = flag.Bool("use_alts", false, "Connection uses ALTS if true (this option can only be used on GCP)")
+	useGoogleDefaultCreds = flag.Bool("use_google_default_creds", false, "Uses google default creds if true")
```

Comment:
My suggestion is ugly here, but in order to be consistent with Java and C++, mind switching the command arg from <bool> "--use_google_default_credentials" to <string> "--custom_credentials_type", and then when use google default credentials when  "google_default_credentials" is passed in as the value?

Doing so would just make it consistent with the other languages (See e.g.https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L782)

Reply:
This indeed makes me feel sad. But done anyway.